ebin/rebar.app

```diff
@@ -52,7 +53,11 @@
                   compiler,
                   crypto,
                   syntax_tools,
-                  tools]},
+                  tools,
+                  diameter,
```

Comment:
Again I forgot. Why do we depend on this? Did we have diameter compiler support or something?